MiniApp.xcodeproj/project.pbxproj

```diff
@@ -258,15 +261,6 @@
 			path = Assets;
 			sourceTree = "<group>";
 		};
-		38920ADD251D9C57004C5DDD /* Config */ = {
-			isa = PBXGroup;
-			children = (
-				38920ADE251D9C57004C5DDD /* Config-debug.xcconfig */,
-				38920ADF251D9C57004C5DDD /* Config-release.xcconfig */,
-			);
-			path = Config;
-			sourceTree = "<group>";
-		};
```

Comment:
this was a duplicated config folder we could find twice in host app file structure